### PR TITLE
Zenoh bridge

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

_Explanation of changes goes here_

Added:
 - MQTT bridge module for stage 1 of zenoh impl
 - Zenoh handler to eventually replace MQTT handler

## Notes

_Any other notes go here_

There is code duplication between mqtt_handler, zenoh_bridge, and zenoh_handler.  I do not care because I fully expected mqtt_handler and zenoh_bridge to be deprecated and eventually deleted.

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: src/zenoh_handler.rs [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+        // if augment HV on, send as such, otherwise start default off
+        let mut last_stat = if self.augment_hv_on {
+            let time = SystemTime::now()
+                .duration_since(UNIX_EPOCH)
+                .unwrap()
+                .as_millis() as u64;
+            warn!("HV status permanently set on!!");
+            if let Err(err) =
+                std::fs::create_dir(format!("{}/event-{}", SAVE_LOCATION.get().unwrap(), time))
```
Comment (caiodasilva2005) [inline src/zenoh_handler.rs:188]: Aren't you already doing this logic in the receive handler of the zenoh processor?

Comment (jr1221) [reply src/zenoh_handler.rs:188]: Yes there is some code copy paste but the reason this exists is because oft he augment HV on flag.  It initializes the library  pretending such a message was received at boot.
